jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/redshift/RedshiftColumnMetadataReader.java

```diff
@@ -0,0 +1,33 @@
+package com.exasol.adapter.dialects.redshift;
+
+import java.sql.Connection;
+import java.sql.Types;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.dialects.JdbcTypeDescription;
+import com.exasol.adapter.jdbc.BaseColumnMetadataReader;
+import com.exasol.adapter.metadata.DataType;
+
+/**
+ * This class implements a Redshift-specific column metadata reader
+ */
+public class RedshiftColumnMetadataReader extends BaseColumnMetadataReader {
+    /**
+     * Create a new instance of a {@link RedshiftColumnMetadataReader}
+     *
+     * @param connection JDBC connection to the remote data source
+     * @param properties user-defined adapter properties
+     */
+    public RedshiftColumnMetadataReader(final Connection connection, final AdapterProperties properties) {
+        super(connection, properties);
+    }
+
+    @Override
+    public DataType mapJdbcType(final JdbcTypeDescription jdbcTypeDescription) {
+        if (jdbcTypeDescription.getJdbcType() == Types.NUMERIC) {
+            return mapJdbcTypeNumericToDecimalWithFallbackToDouble(jdbcTypeDescription);
```

Comment:
`super` is missing.

Reply:
Explicit `super` is not necessary here. Only in the `else` since we would otherwise create an endless loop.